core/EE_Load_Textdomain.core.php

```diff
@@ -27,7 +27,7 @@ class EE_Load_Textdomain extends EE_Base
      */
     public static function load_textdomain()
     {
-        self::_maybe_get_langfile();
+        self::_maybe_get_langfiles();
```

Comment:
sorry, but you can't just rename a public file since it's possible that someone might be using this in their own addon or theme or whatever. It's unlikely that this class is being used by anyone else but it wouldn't be the first time I've thought that and been wrong if someone was using it.

So what you need to do is keep the existing function declarations as is and then just add your new functions to the class.
If the old functions are completely unwanted, then they can be forwarded to the new functions (ie: calling A calls B) and doing it wrong notices can be added to inform devs that the old function is deprecated and scheduled for eventual removal.

Reply:
ug... totally read this wrong.